lib/dock_health_api/crud/update.rb

```diff
@@ -2,10 +2,15 @@ module DockHealthApi
   module Crud
     module Update
       def update(**params)
+        if params.key?(:organizationId) && params.key?(:userId)
```

Comment:
I think you can just override the update method for the Organization::User class instead of this